packages/gatsby-image/src/index.js

```diff
@@ -437,6 +437,7 @@ class Image extends React.Component {
       itemProp,
       loading,
       draggable,
+      tagProps = {},
```

Comment:
Let's be extra explicit here

```suggestion
      imageTagProps = {},
```

Reply:
It's not for the `img` element though? It's for the `div`(default tag) wrapper element. My review suggested `wrapperAttrs` or `attrProps` which is more clear what the purpose of this props is(attributes for passing to the top-level element). We have lots of "image"/"tag"/"props" keywords in the file already, so I definitely think it's more clearer in all that keyword noise to be more specific it's about attributes.